src/interval.jl

```diff
@@ -167,24 +174,21 @@ end
 Base.isless(a::AbstractInterval, b) = LeftEndpoint(a) < b
 Base.isless(a, b::AbstractInterval) = a < LeftEndpoint(b)
 
-less_than_disjoint(a::AbstractInterval, b) = RightEndpoint(a) < b
-less_than_disjoint(a, b::AbstractInterval) = a < LeftEndpoint(b)
-
-function Base.:isless(a::AbstractInterval, b::AbstractInterval)
+function Base.isless(a::AbstractInterval, b::AbstractInterval)
     return LeftEndpoint(a) < LeftEndpoint(b)
 end
 
-function less_than_disjoint(a::AbstractInterval, b::AbstractInterval)
+isless_disjoint(a::AbstractInterval, b) = RightEndpoint(a) < b
+isless_disjoint(a, b::AbstractInterval) = a < LeftEndpoint(b)
+
+function isless_disjoint(a::AbstractInterval, b::AbstractInterval)
     return RightEndpoint(a) < LeftEndpoint(b)
 end
 
-greater_than_disjoint(a, b) = less_than_disjoint(b, a)
-
 """
     ≪(a::AbstractInterval, b::AbstractInterval) -> Bool
-    less_than_disjoint(a::AbstractInterval, b::AbstractInterval) -> Bool
```

Comment:
Perhaps just change the reference here to `isless_disjoint`?

Reply:
I ended up removing the reference here as there isn't a `isgreater_disjoint` for the other docstring. Additionally, this specific docstring will only appear with `help?> ≪` and not with `help?> isless_disjoint`